ci/appveyor/package.cfg

```diff
@@ -4,16 +4,21 @@ packages = {pkg_prefix}libsodium
 
 filename_stub = aeternity-windows
 
-display_name = aeternity for Windows
+display_name = Aeternity for Windows
 
-description = The full aeternity node for Windows.
+description = The full Aeternity node for Windows.
 
 version = 1.5.0-alpha.1
 
 url = https://github.com/aeternity/aeternity
 
 launchers =
     aeternity.desktop
+    keys_gen.util
+    check_config.util
+    export_chain.util
+    messages_hash.util
+    peer_key.util
```

Comment:
Other list prone to get out of sync ... is there any automated check? Or a smoke test that exercise at least once each of the lists?

Reply:
I see test.bat commented in CI.

Reply:
Wrt extensions themselves, I see two ways that those could be tested both on Win and Linux/macOS:
* Integration tests (`make ct`)
* Package smoke test ([`make python-release-test`](https://github.com/aeternity/aeternity/blob/e58cfd5c5677f67828d67c1e8364b7b0b1b9446b/.circleci/config.yml#L130-L140))

Reply:
I added a python-based test.